Assets/MRTK/SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -2580,8 +2580,13 @@ private void OnPointerUp(MixedRealityPointerEventData eventData)
             }
         }
 
-        private void OnPointerDown(MixedRealityPointerEventData eventData)
+        void IMixedRealityPointerHandler.OnPointerDown(MixedRealityPointerEventData eventData)
         {
+            // Other option is to check propagation phase
```

Comment:
If you want to mention the other option (which I think is optional) it would be good to also give an example.